app/graphql/types/proxy_file_set_type.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+class Types::ProxyFileSetType < Types::BaseObject
+  implements Types::Resource
+  def viewing_hint; end
+
+  def label
+    object.label.try(:first)
+  end
+
+  # There is no thumbnail for an audio file, so this has to return blank. The
```

Comment:
some audio files have thumbnails. but not the kind that will be proxied I guess.

Reply:
Do you want me to change this somehow?

Reply:
Yeah, I guess it would be misleading if someone unfamiliar with the different collections in figgy read it. Maybe just more specific like "the audio files we plan to proxy for playlists won't have thumbnails, so [etc]"

Reply:
That look better?

Reply:
yes thank you!